private/rules/jvm_import.bzl

```diff
@@ -15,27 +15,38 @@ def _jvm_import_impl(ctx):
 
     injar = ctx.files.jars[0]
     outjar = ctx.actions.declare_file("processed_" + injar.basename, sibling = injar)
-    ctx.actions.run_shell(
-        inputs = [injar] + ctx.files._add_jar_manifest_entry,
+    args = ctx.actions.args()
+    args.add_all(["--source", injar, "--output", outjar])
+    args.add_all(["--manifest-entry", "Target-Label:{target_label}".format(target_label = ctx.label)])
```

Comment:
Can this be combined with the call below so we only call add_jar_manifest_entry once?

Reply:
Desirable as that may be, the stamped jar and the compile jar are used for different purposes. The output jar will be supplied as a transitive dependency and is included in the list of runtime jars.

The compile jar is just used by javac for compilation. Normally, bazel uses `ijar` to generate this, and that strips out the bodies of methods, leaving only the ABI in place. For this case, we could remove everything that wasn't the class files, and reduce the manifest down to indicate whether or not this is a multi-release jar. In previous reviews, you've asked for the manifest to be left intact, which is done, other than stripping the `Class-Path`, since that's the thing that causes bazel to the `bad path element` warnings this PR was designed to remove :)